app/models/customer.rb

```diff
@@ -22,4 +22,8 @@ def successful_transactions_count
     transactions.where('result = ?', 0)
     .count
   end
+
+  def name
+    "#{first_name} #{last_name}"
```

Comment:
Great job adding this down to the method and combining the names into a full name for a customer.